packages/shared/src/utils/os/os_detect.ts

```diff
@@ -40,8 +51,8 @@ export const OSDetect = () => {
     }
     if (typeof navigator !== 'undefined' && navigator.platform) {
         return Object.keys(systems)
-            .map(os => {
-                if (systems[os].some(platform => navigator.platform === platform)) {
+            .map((os: string) => {
```

Comment:
```suggestion
            .map((os) => {
```
@niloo-fs could you please check if map() infers the type of `os` without explicitly declaring it?🙂 It worked for me)